docs/user/alerting/action-types.asciidoc

```diff
@@ -27,6 +27,17 @@ a| <<servicenow-action-type, ServiceNow>>
 
 | Create an incident in ServiceNow.
 
+a| <<jira-action-type, Jira>>
+
+| Create an incident in Jira.
+
+a| <<resilient-action-type, IBM Resilient>>
+
+| Create an incident in IBM Resilient.
```

Comment:
**nit:** The list was in alphabetical order. I'm not sure if that's a standard for docs but thought I'd mention it.

Reply:
@mikecote I agree.  The list should be in alphabetical order.

Reply:
The actions should also appear in alphabetical order in the docs table of contents.